src/main/java/org/bytedeco/javacpp/Pointer.java

```diff
@@ -859,11 +859,61 @@ public int sizeof() {
 
     private native ByteBuffer asDirectBuffer();
     /**
-     * Returns a ByteBuffer covering the memory space from this.position to this.limit.
-     * If limit == 0, it uses position + 1 instead. The way the methods were designed
-     * allows constructs such as {@code this.position(13).limit(42).asByteBuffer()}.
```

Comment:
Let's not remove the correct information that is already there. Please only add new information.

Reply:
Did you want that precise verbiage? I updated some pieces to match the documentation found in the ByteBuffer classes.

Reply:
No, your update looks fine! Thanks